trulens_eval/examples/expositional/models/azure_openai.ipynb

```diff
@@ -29,7 +29,7 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #18.            return float(re_0_10_rating(self.endpoint.run_me(
</pre><p>This isn't that ideal because the functions re_0_10_rating and endpoint.run_me aren't that well documented or understood. Could we refactor these so that we create a bit more of an externally facing function that is better documented and more understandable?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/847/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Refactored and added a new user-friendly method `generate_score` that handles these internal calls.

Also added `generate_score_and_reasons` method for COT methods.